server/.env.example

```
@@ -4,3 +4,4 @@ DISCORD_TOKEN=
 STAFFAPPS_GUILD_ID=
 STAFFAPPS_CATEGORY_ID=
 STAFFAPPS_APPLICATION_CHANNEL_ID=
+CORS_VALID_DOMAINS=
```

```suggestion
CORS_VALID_DOMAINS=slmn.gg,localhost
```
Example env file can have these here by default, they'll always be needed and can fallback to these fine